cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -1022,6 +1030,33 @@ message RefreshLockResponse {
   cs3.types.v1beta1.Opaque opaque = 2;
 }
 
+message UnlockAndRelockRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
+  // REQUIRED.
+  // The reference on which the lock should be UnlockedAndRelocked
+  Reference ref = 2;
+  // REQUIRED.
+  // The lock metadata.
+  Lock lock = 3;
+  // REQUIRED.
+  // The existing lock id
+  string oldlock_id = 4;
+}
+
+message UnlockAndRelockResponse {
+  // REQUIRED.
+  // The response status.
+  cs3.rpc.v1beta1.Status status = 1;
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 2;
+  // REQUIRED.
+  // The lock metadata
+  Lock lock = 3;
```

Comment:
Why would you need to return the new lock data structure on response? After all this operation is similar to RefreshLock, the server could just say "done" or "failed".

Reply:
ok, you are correct. The new refresh metadata is already available for the caller. will change.